template-ui/src/components/CardBody.vue

```diff
@@ -1,19 +1,21 @@
 <template>
-  <div>
-    <p class="text-uppercase text-info mb-1">
+  <div class="card-content">
+    <p class="text-uppercase text-info mb-1 card-type text-truncate">
```

Comment:
This new card-type class isn't used.

```suggestion
    <p class="text-uppercase text-info mb-1 text-truncate">
```